2.3.1 release build issue

👋 trying to build the latest release, but run into some build issue. The error log is as below:


<details>
<summary>error build log</summary>

```
  /tmp/opencolorio-20231211-6437-rqqe1t/OpenColorIO-2.3.1/src/OpenColorIO/ConfigUtils.cpp:6:10: fatal error: pystring/pystring.h: No such file or directory
      6 | #include "pystring/pystring.h"
        |          ^~~~~~~~~~~~~~~~~~~~~
  compilation terminated.
```

</details>

relates to Homebrew/homebrew-core#156951

--- a/src/OpenColorIO/ConfigUtils.cpp
+++ b/src/OpenColorIO/ConfigUtils.cpp
@@ -1,9 +1,10 @@
 // SPDX-License-Identifier: BSD-3-Clause
 // Copyright Contributors to the OpenColorIO Project.
 
+#include <pystring.h>
+
 #include "ConfigUtils.h"
 #include "MathUtils.h"
-#include "pystring/pystring.h"
 #include "utils/StringUtils.h"
 
 namespace OCIO_NAMESPACE

